spec.md

```diff
@@ -1847,6 +1847,33 @@ To disallow infinitely recursive hardware, modules cannot contain instances of
 itself, either directly, or indirectly through instances of other modules it
 instantiates.
 
+## Remote Instances
+
+FIRRTL modules may also be remotely instantiated.  A remotely instantiated
+module will cause the instantiation to be emitted outside its instantiating
```

Comment:
```suggestion
module will cause the instantiation to be emitted outside its instantiating
```

is there any "may" or "will" here? are there conditions where we might not choose to do that sort of lowering?

Reply:
conversely, there are times where we choose to convert things to binds...